client/my-sites/plugins-wpcom/plugin-types/business-plugin.jsx

```diff
@@ -11,12 +11,15 @@ export const BusinessPlugin = React.createClass( {
 			name,
 			plan,
 			onClick = noop,
-			supportLink
+			supportLink,
+			isExternalLink = true,
 		} = this.props;
 
+		const target = isExternalLink ? '_blank' : '_self';
```

Comment:
I'm still curious why we need this to be explicitly declared in the data structure and props. Shouldn't it be clear from the URL if it's external? In other words, won't all internal links be invalid URLs (missing the protocol and host) while external URLs will necessarily be fully valid?

```js
const target = isFullyFormedUrl( link )
	? '_blank'
	: '_self';
```